source/agora/common/crypto/Schnorr.d

```diff
@@ -421,6 +421,14 @@ public Sig multiSigCombine (S)(const S sigs) nothrow @nogc @safe
     return Sig(sum_R, sum_s);
 }
 
+/// multi-sig add a single signature
+public Sig multiSigAdd (const ref Sig multi_sig, const Sig sig) nothrow @nogc @safe
```

Comment:
How is this different to `multiSigCombine`? It seems to do the exact same thing.

Reply:
Technically you could do `multiSigCombine(sig.only.chain(sig2))` although it doesn't look pretty.

Reply:
It is for adding a single signature to a signature which may be multiple signatures combined.

Reply:
Actually your right. I'll update it to use the same.

Reply:
Updated.